packages/clusview/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+No idea what this is :(
+"""
```

Comment:
This is the root package. It is not used directly in the imports (assuming the `venv` is located in the root too). This should just describe the idea of the whole tool. Something along the lines of the description you can find at the GitHub's repo page.

> Configure, optimize and visualize document clustering.